improc/alignment.py

```diff
@@ -350,7 +350,14 @@ def _align_swarp( self, image, target, sources, target_sources ):
             #  For most images I've seen, no image
             #  will have a pixel with noise above 100000,
             #  hence the 1e-10.
-            warpedim.flags[ warpedim.weight < 1e-10 ] = 1
+            # WH: to get the out of bounds pixels from the alignment
+            #  flag all such pixels that were not already flagged bad
```

Comment:
You should feel free to get rid of old code without leaving comments, unless there's still some debate about removing it or not. This is why we do code review!